src/@vultr/vultr-node.d.ts

```diff
@@ -0,0 +1,113 @@
+declare module "@vultr/vultr-node" {
+  function initialize(options: { apiKey: string }): Vultr;
+
+  interface Vultr {
+    dns: {
+      listRecords: (parameters: {
+        "dns-domain": string;
+        per_page: number;
+        cursor?: string;
+      }) => Promise<RecordListWrapper>;
+
+      createRecord: (parameters: {
+        "dns-domain": string;
+        name: string;
+        type: "A" | "CNAME";
+        data: string;
+      }) => Promise<RecordWrapper>;
+
+      deleteRecord: (parameters: {
+        "dns-domain": string;
+        "record-id": string;
+      }) => Promise<void>;
+    };
+
+    instances: {
+      createInstance: (parameters: {
+        region: string;
+        plan: string;
+        os_id: string;
```

Comment:
nit: is `os_id` actually a number (eg https://www.vultr.com/api/#tag/os/operation/list-os)? 

Is there value in defining an enum for this one, even if "partial" based on OS's we're most likely to try? Plus a code comment to jog our memory of which is which - might mean slightly smarter autocomplete suggestions in our code editors downstream and save some external lookups?

Eg: 
```ts
  // 1743 = ubuntu, 2076 = alpine, etc
  os_id: 1743 | 2076 | number;
```

Reply:
done in [0112c7b](https://github.com/theopensystemslab/vultr-action/pull/2/commits/0112c7b921b77b6bda91bf9b11a4fd344a4b8020) :)

Reply:
I'm into it! Thanks for explanation too